src/browser/predicates.js

```diff
@@ -36,6 +36,10 @@ function urlIsBlacklisted(item, settings) {
   return urlIsOnAList(item, settings, 'blacklist');
 }
 
+function urlIsNotBlacklisted(item, settings) {
+  return !urlIsBlacklisted(item, settings);
```

Comment:
Rather than have this indirection, just replace `!urlIsBlacklisted(...)` with `!urlIsOnAList(item, settings, 'blacklist')` and get rid of the `urlIsBlacklisted` function.